src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,123 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final String INSERT_FORMAT_REQUEST =
+            "INSERT INTO manufacturers(name, country) VALUES(?, ?)";
+    private static final String GET_REQUEST =
+            "SELECT * FROM manufacturers WHERE id = ?";
+    private static final String GET_ALL =
+            "SELECT * FROM manufacturers WHERE is_deleted = false";
```

Comment:
```suggestion
            "SELECT * FROM manufacturers WHERE is_deleted = FALSE";
```
please fix the same issue in all places